tests/testapps/CGCatalog/CGCatalog/CGPathAddArcViewController.m

```diff
@@ -0,0 +1,76 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import "CGPathAddArcViewController.h"
+#import "CGDrawView.h"
+
+@implementation CGPathAddArcViewController
+
+- (id)initWithDrawingOptions:(CGDrawOptions*)options {
+    self = [super initWithDrawingOptions:options];

```

Comment:
Yup, this contains the line color, dash pattern, width, etc.. that each CG sample view uses.

---
In reply to: [88085579](https://github.com/Microsoft/WinObjC/pull/1384#discussion_r88085579) [](ancestors = 88085579)